lug/worker-script/mirrorz.py

```diff
@@ -87,7 +87,28 @@ def main():
         mirrors.append(mirror)
 
     mirrorz["mirrors"] = mirrors
-    print(json.dumps(mirrorz))
+    mirrorz["extension"] = "D"
+    mirrorz["endpoints"] = [{
+        "label": "sjtug-" + site["tag"],
```

Comment:
label should not contain `-`.

This can work. But if users are using `https://sjtug-siyuan.mirrors.cernet.edu.cn` as their preferred endpoint, they wont get desired result as the request label would be split into `["sjtug", "siyuan"]` and wont match this endpoint.